cluster-up/cluster/kind/common.sh

```diff
@@ -12,6 +12,10 @@ CONFIG_WORKER_CPU_MANAGER=${CONFIG_WORKER_CPU_MANAGER:-false}
 # avaliable value: ipv4, ipv6, dual
 IPFAMILY=${IPFAMILY}
 
+# setup the port mapping for kind cluster, this is needed for some e2e tests
+# KIND_PORT_MAPPING=cluster_port:host_port e.g. KIND_PORT_MAPPING=30001:30002
+KIND_PORT_MAPPING=${KIND_PORT_MAPPING}
```

Comment:
Can this take a comma separated list of port mappings or is only one port mapping allowed?

Reply:
Only allow one port mapping. One port mapping is enough for the e2e tests.